draft-pauly-privacypass-auth-scheme.md

```diff
@@ -329,17 +331,18 @@ protocol may separately involve user interaction if the client needs to be newly
 validated.
 
 The use of interactive tokens can add user-perceivable latency, since such
-tokens cannot be pre-fetched. Origins SHOULD NOT block useful work on token
+tokens cannot be pre-fetched. Origins need not block useful work on token
 authentication. Instead, token authentication can be used in similar ways to
 CAPTCHA validation today, but without the need for user interaction. If issuance
 is taking a long time, a website could show an indicator that it is waiting,
 or fall back to another method of user validation.
 
-If an origin is requesting an unexpected number of tokens, such as requesting
-token authentication more than once for a single website load, it can indicate
-that the server is not functioning correctly, or is trying to attack or overload
-the client or issuance servers. In such cases, the client SHOULD ignore
-redundant token challengers, or else alert the user.
+An origin MUST NOT issue more than one challenge per client request. If an
+origin issues a large number of challenges, such as more than once for each
+request, this can indicate that the origin is either not functioning correctly
+or is trying to attack or overload the client or issuance server. In such cases,
+a client MUST ignore redundant token challenges for the same request and SHOULD
+alert the user if possible.
```

Comment:
```suggestion
An origin MUST NOT issue more than one interactive challenge for a given token type
and issuer per client request. If an origin issues a large number of challenges, such
as more than once for each request, this can indicate that the origin is either not
functioning correctly or is trying to attack or overload the client or issuance server.
In such cases, a client MUST ignore redundant token challenges for the same
request and SHOULD alert the user if possible.

Origins MAY include multiple challenges, where each challenge refers to a different issuer
or a different token type, to allow clients to choose a preferred issuer or type.
```